src/Physics/Evaluate_friction_law.f90

```diff
@@ -59,14 +59,14 @@ MODULE Eval_friction_law_mod
   END INTERFACE
   !---------------------------------------------------------------------------!
   PUBLIC  :: Eval_friction_law
-  PRIVATE :: updateStateVariable
-  PRIVATE :: IterativelyInvertSR
-  PRIVATE :: no_fault
-  PRIVATE :: Linear_slip_weakening_bimaterial
-  PRIVATE :: Linear_slip_weakening_TPV1617
-  PRIVATE :: ImposedSlipRateOnDRBoundary
-  PRIVATE :: rate_and_state
-  PRIVATE :: rate_and_state_vw
+!  PRIVATE :: updateStateVariable
```

Comment:
remove?

Reply:
Could do so, but currently it is useful to have this as reference in the working tree.